templates/kubernetes/base/kustomization.yml

```diff
@@ -14,6 +14,9 @@ configMapGenerator:
     - DATABASE_HOST=database.<% .Name %>
     - DATABASE_PORT=<%if eq (index .Params `database`) "postgres" %>5432<% else if eq (index .Params `database`) "mysql" %>3306<% end %>
     - DATABASE_NAME=<% index .Params `databaseName` %>
+    - CACHE_ENGINE=<% index .Params `cacheStore` %>
```

Comment:
If cache engine is empty let's exclude these vars

Reply:
use "none"